addons/binding/org.openhab.binding.mcp23017/README.md

```diff
@@ -60,6 +60,8 @@ Configuration with default_state and pull_mode:
 Thing mcp23017:mcp23017:chipA  "MCP23017 chip A" [address=20,bus=1] {
     Type output_pin : output#A0 [default_state="HIGH"]
     Type output_pin : output#A1 [default_state="LOW"]
+    Type output_pin : output#A2 [active_low="y"]
+    Type output_pin : output#A2 [default_state="LOW", active_low="y"]
```

Comment:
I think you should name it `inverse` and make it boolean. That seems more logical. The fact that you have to explain it in the title by also stating "invert logic" possible already says it's not clear when you name it "active_low"

Reply:
Thank you for your Review. I choose "active_low" as name bebause its a "wellknown name" for this logic. As far as I know, the GPIO-Addon and the the linux-kernel itselfs name it so.

Reply:
I'm familiar with the active low/high terminology. With a term like active low/high it is immediately clear what the behavior is. With inverse you need to remember/lookup what the default behavior is.

See also [Logic level](https://en.wikipedia.org/wiki/Logic_level) on Wikipedia.